src/DataTable/components/Pagination.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+// Actions
+import { changePage } from '../slices/dataTable'
+
+const Pagination = () => {
+  const dispatch = useDispatch();
+  const { currentPageNumber, totalNumberOfPages, rowsPerPage } = useSelector((state) => ({
+      currentPageNumber: state.dataTable.currentPageNumber,
+      totalNumberOfPages: state.dataTable.totalNumberOfPages,
+      rowsPerPage: state.dataTable.rowsPerPage
+  }));
+
+  const previousPageDisabled = useMemo(() => {
+    return currentPageNumber === 0;
+  }, [currentPageNumber]);
+  
+  const renderesCurrentPageNumber = useMemo(() => {
+    return totalNumberOfPages === 0 ? 0 : currentPageNumber + 1;
+  }, [currentPageNumber, totalNumberOfPages]);
+  
+  const nextPageDisabled = useMemo(() => {
+    return renderesCurrentPageNumber === totalNumberOfPages;
+  }, [renderesCurrentPageNumber, totalNumberOfPages]);
```

Comment:
All these 3 useMemos should be computed in the slice/selectors. It doesn't make sense to compute that in component, and it would greatly simplify things component's code